features/utils-wp.feature

```diff
@@ -79,6 +86,18 @@ Feature: Utilities that depend on WordPress code
       """
     And save STDOUT as {DEFAULT_STDOUT}
 
+    When I run `wp --require=table_names.php get_table_names --all-tables --views-only`
+    Then STDOUT should contain:
+      """
+      posts_view
```

Comment:
This should also ensure the tables are not displayed. Otherwise, the flag might just not do anything and have the test still pass.